src/smart-components/access-management/UserGroupsTable.tsx

```diff
@@ -162,25 +190,79 @@ const UserGroupsTable: React.FunctionComponent<UserGroupsTableProps> = ({
   );
 
   return (
-    <DataView ouiaId={ouiaId} selection={selection}>
-      <DataViewToolbar
-        ouiaId={`${ouiaId}-header-toolbar`}
-        bulkSelect={
-          <BulkSelect
-            isDataPaginated
-            pageCount={groups.length}
-            selectedCount={selected.length}
-            totalCount={totalCount}
-            pageSelected={pageSelected}
-            pagePartiallySelected={pagePartiallySelected}
-            onSelect={handleBulkSelect}
+    <Fragment>
+      {isDeleteModalOpen && (
+        <WarningModal
+          ouiaId={`${ouiaId}-remove-user-modal`}
+          isOpen={isDeleteModalOpen}
+          title={
+            <FormattedMessage
+              {...messages.deleteUserGroupModalTitle}
+              values={{
+                count: currentGroups.length,
+                plural: currentGroups.length > 1 ? intl.formatMessage(messages.groups) : intl.formatMessage(messages.group),
+              }}
+            />
+          }
+          withCheckbox
+          checkboxLabel={intl.formatMessage(messages.understandActionIrreversible)}
+          confirmButtonLabel={intl.formatMessage(messages.remove)}
+          confirmButtonVariant={ButtonVariant.danger}
+          onClose={() => setIsDeleteModalOpen(false)}
+          onConfirm={() => {
+            handleDeleteGroups(currentGroups);
+          }}
+        >
+          <FormattedMessage
+            {...messages.deleteUserGroupModalBody}
+            values={{
+              b: (text) => <b>{text}</b>,
+              count: currentGroups.length,
+              plural: currentGroups.length > 1 ? intl.formatMessage(messages.groups) : intl.formatMessage(messages.group),
+              name: currentGroups[0]?.name,
+            }}
           />
-        }
-        pagination={React.cloneElement(paginationComponent, { isCompact: true })}
-      />
-      <DataViewTable variant="compact" aria-label="Users Table" ouiaId={`${ouiaId}-table`} columns={COLUMNS} rows={rows} />
-      <DataViewToolbar ouiaId={`${ouiaId}-footer-toolbar`} pagination={paginationComponent} />
-    </DataView>
+        </WarningModal>
+      )}
+      <DataView ouiaId={ouiaId} selection={selection}>
+        <DataViewToolbar
+          ouiaId={`${ouiaId}-header-toolbar`}
+          bulkSelect={
+            <BulkSelect
+              isDataPaginated
+              pageCount={groups.length}
+              selectedCount={selected.length}
+              totalCount={totalCount}
+              pageSelected={pageSelected}
+              pagePartiallySelected={pagePartiallySelected}
+              onSelect={handleBulkSelect}
+            />
+          }
+          actions={
+            <div data-ouia-component-id={`${ouiaId}-actions-dropdown`}>
+              <ActionsColumn
```

Comment:
Let's use newer functionality from DataView https://react-data-view-pr-data-view-131.surge.sh/extensions/data-view/components#actions-example this would allow us to keep actions button visible all the times.

Reply:
@karelhala The `<ResponsiveActions>` component doesn't currently support disabling actions through any props like `isDisabled` or `disabled`. I'll switch to using it once that functionality is added. We need that here in the case that no rows are selected or any rows are `system` or `platform_default`.

Reply:
All other feedback should be resolved :white_check_mark:

Reply:
You can disable each action by setting `isDisabled` on ResponsiveAction
```
<ResponsiveActions breakpoint="lg">
    <ResponsiveAction isPersistent isDisabled>
        Persistent Action
    </ResponsiveAction>
    <ResponsiveAction isDisabled isPinned variant='secondary'>
        Pinned Action
    </ResponsiveAction>
    <ResponsiveAction>
        Overflow Action
    </ResponsiveAction>
  </ResponsiveActions>
```

Reply:
@karelhala It's working when the `ResponsiveAction` is marked `isPersistent`, but for actions within the dropdown, `isDisabled` isn't disabling them.